<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>960</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lens Simulator</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Open"/>
   </widget>
   <widget class="QMenu" name="menuH_elp">
    <property name="title">
     <string>H&amp;elp</string>
    </property>
    <addaction name="action_About"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>View</string>
    </property>
    <widget class="QMenu" name="menuShow">
     <property name="title">
      <string>Show</string>
     </property>
     <addaction name="action_Entrance_Exit_Pupils"/>
     <addaction name="action_Focal_Points_and_Principal_Planes"/>
    </widget>
    <addaction name="action_Editor"/>
    <addaction name="action_Point_Imaging"/>
    <addaction name="action_Console"/>
    <addaction name="menuShow"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menuWindow"/>
   <addaction name="menuH_elp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidgetEditorView">
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QTableView" name="tableViewLensEditor"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidgetConsole">
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetConsoleContents">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTextEdit" name="textEditConsole"/>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="action_Open">
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="action_Editor">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Editor</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="action_Point_Imaging">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Point Imaging</string>
   </property>
  </action>
  <action name="action_Console">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Console</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="action_Entrance_Exit_Pupils">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Entrance/Exit Pupils</string>
   </property>
  </action>
  <action name="action_Focal_Points_and_Principal_Planes">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Focal Points and Principal Planes</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
